cosmo_tester/test_suites/manager_of_managers/regional_cluster.py

```diff
@@ -0,0 +1,974 @@
+                self.deployment_id
+            )
+        )
+
+        backup_params = {
+            'snapshot_id': self.deployment_id,
+            'backup_params': []
+        }
+        self.cfy.executions.start(
+            'backup', '-d', self.deployment_id,
+            '-p', json.dumps(backup_params)
+        )
+        self.logger.info('Backup completed successfully')
+
+    def execute_hello_world_workflow(self, workflow_id):
+        self.logger.info(
+            'Executing workflow {0} on deployment {1} '
+            'on a Regional cluster...'.format(
+                workflow_id,
+                constants.HELLO_WORLD_DEP)
+        )
+        workflow_params = {
+            'workflow_id': workflow_id,
+            'deployment_id': constants.HELLO_WORLD_DEP,
+            'tenant_name': constants.TENANT_1
+        }
+
+        self.cfy.executions.start([
+            'execute_workflow',
+            '-d', self.deployment_id,
+            '-p', json.dumps(workflow_params)
+        ])
+        self.logger.info(
+            'Successfully executed workflow {0} on deployment {1} '
+            'on a Regional cluster'.format(
+                workflow_id,
+                constants.HELLO_WORLD_DEP)
+        )
+
+
+class FixedIpRegionalCluster(AbstractRegionalCluster):
+    TRANSFER_AGENTS = False
+    RESOURCE_POOL1 = [
+        {
+            'ip_address': '10.0.0.50',
+            'hostname': 'db_worker_'
+        },
+        {
+            'ip_address': '10.0.0.51',
+            'hostname': 'db_worker_2'
+        }
+    ]
+    RESOURCE_POOL2 = [
+        {
+            'ip_address': '10.0.0.52',
+            'hostname': 'queue_worker_1'
+        },
+        {
+            'ip_address': '10.0.0.53',
+            'hostname': 'queue_worker_2'
+        }
+    ]
+    RESOURCE_POOL3 = [
+        {
+            'ip_address': '10.0.0.54',
+            'hostname': 'seed_worker_1'
+        },
+        {
+            'ip_address': '10.0.0.55',
+            'hostname': 'seed_worker_2'
+        }
+    ]
+    RESOURCE_POOL4 = [
+        {
+            'ip_address': '10.0.0.56',
+            'hostname': 'additional_worker_1'
+        },
+        {
+            'ip_address': '10.0.0.57',
+            'hostname': 'additional_worker_2'
+        }
+    ]
+
+    @property
+    def network_inputs(self):
+        return {
+            # Only relevant when working with the Private Fixed IP paradigm.
+            # See more in private_fixed_ip.yaml
+            'queue-infrastructure--'
+            'resource_pool': self.RESOURCE_POOL1,
+            'database-infrastructure--'
+            'resource_pool': self.RESOURCE_POOL2,
+            'seed-worker-infrastructure--'
+            'resource_pool': self.RESOURCE_POOL3,
+            'additional-workers-infrastructure--'
+            'resource_pool': self.RESOURCE_POOL4,
+        }
+
+    @property
+    def inputs(self):
+        # To see explanations of the following inputs, see
```

Comment:
Is this still relevant?

Reply:
Updated

Reply:
If we don't use the inputs from 'AbstractRegionalCluster', this need to be updated

Reply:
Updated,